tests/tiling/launchers/README.txt

```diff
@@ -0,0 +1,28 @@
+Run dispatcher.sh from the directory /path/to/seigen/tests/tiling as:
+
+    node=X mode=Y ./launchers/dispatcher.sh
+
+Where X can be:
+
+    * erebus
+    * cx1-ivy
+    * cx1-haswell
+    * cx2-westmere
+    * cx2-sandyb
+    * cx2-haswell
+    * cx2-broadwell
+
+And Y can be:
+
+    * singlenode : to run the test suite on the single node Y
```

Comment:
Shouldn't this be "on the single X node"? And "on a cluster of X nodes" for the line below?